sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/IResourceSkusOperations.cs

```diff
@@ -46,7 +49,7 @@ public partial interface IResourceSkusOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<IPage<ResourceSku>>> ListWithHttpMessagesAsync(string filter = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<IPage<ResourceSku>>> ListWithHttpMessagesAsync(string filter = default(string), string includeExtendedLocations = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
does something like this need customization?

Reply:
Yes.

Reply:
need customization per Mark